pom.xml

```diff
@@ -1735,7 +1735,7 @@ under the License.
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-javadoc-plugin</artifactId>
-					<version>3.1.1</version><!--$NO-MVN-MAN-VER$-->
+					<version>2.9.1</version><!--$NO-MVN-MAN-VER$-->
```

Comment:
This means that we cannot build the JavaDocs with Java 11, right?

Reply:
Correct.